pkg/testframework/config.go

```diff
@@ -39,3 +45,90 @@ func GetConfig() (*rest.Config, error) {
 
 	return nil, fmt.Errorf("could not locate a kubeconfig")
 }
+
+// Reset brand value in place before run of test
+func PutBackBrand(t *testing.T, client *Clientset, originalBrand v1.Brand) {
+	err := SetBrand(client, originalBrand)
+	if err != nil {
+		t.Fatalf("Could not reset brand back to starting value %v", err)
+	}
+}
+
+// Reset the original cm stored in the openshift-config-managed namespace, if it exist
+func PutBackManagedConfigMap(t *testing.T, client *Clientset, ns string, cm *corev1.ConfigMap) {
+	err := client.ConfigMaps(ns).Delete(consoleapi.OpenShiftConsoleConfigMapName, &metav1.DeleteOptions{})
+	if err != nil {
+		t.Fatalf("Could not delete config map %s in namespace %s , %v", cm.Name, ns, err)
+	}
+	// Check that there was a valid config map originally (use name as the validation)
+	if cm.Name != "" {
+		// Need to clear the resource version
+		cm.ResourceVersion = ""
+		_, err = client.ConfigMaps(ns).Create(cm)
+		if err != nil {
+			t.Fatalf("Could not recreate original config map %s in namespace %s , %v", cm.Name, ns, err)
+		}
+	}
+}
+
+// Set Brand on the config
```

Comment:
So I think this naming is confusing.  `SetBrand()` is for the `operator-config`, but `GetBrand()` is for the `console-configmap`.  The `func` names are symetrical, the underlying resources are not.